pdns/namespaces.hh

```diff
@@ -21,7 +21,6 @@
  */
 #pragma once
 
-#include <boost/tuple/tuple.hpp>
 #include <boost/format.hpp>
```

Comment:
It would be nice to get rid of the remaining boost includes as well here and move them to the specific places where they are needed. But that 's likely better done in a separate PR.